src/useDraggable.ts

```diff
@@ -44,17 +38,17 @@ export default function useDraggable() {
     }
 
     const onDrag = (e: MouseEvent) => {
-      track.scrollTo(trackInitialX - e.clientX + mouseInitialX, 0)
+      track.scrollTo((trackInitialX || 0) - e.clientX + (mouseInitialX || 0), 0)
       e.preventDefault()
       e.stopPropagation()
       return false
     }
 
-    const onMouseUp = function () {
-      track.style.scrollBehavior = null
+    const onMouseUp = function (this: any) {
```

Comment:
`this` can't be an argument since the `removeEventListener` has to refer to this function. I don't think we need the `noImplicitThis` rule.

Reply:
No worries I have reverted changes to this file to what the upstream was. With `noImplicitThis` removed no issues are flagged.